hash/pv1000.xml

```diff
@@ -181,4 +181,182 @@ Undumped carts:
 		</part>
 	</software>
 
+	<!-- Homebrew -->
+
+	<software name="aerial">
+		<description>Aerial</description>
+		<year>2022</year>
+		<publisher>Inufuto</publisher>
+		<part name="cart" interface="pv1000_cart">
+			<dataarea name="rom" size = "0x4000">
+				<rom name="aerial.rom" size="0x4000" crc="9d3ddb26" sha1="00630321eef6237501ce099655f3a09cc651d516" offset="0"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="antair">
+		<description>Antair</description>
+		<year>2024</year>
+		<publisher>Inufuto</publisher>
+		<part name="cart" interface="pv1000_cart">
+			<dataarea name="rom" size = "0x4000">
+				<rom name="antiair.rom" size="0x4000" crc="1478db1a" sha1="b0c640776dcdb79db8a7cc8b2387a443182e1036" offset="0"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
This is supposed to be AntiAir: http://inufuto.web.fc2.com/8bit/antiair/

Reply:
romset ID should also be `antiair`, for consistency with `coleco_homebrew.xml`

https://github.com/mamedev/mame/blob/bfb41f8322d93d8a976b4f9e85f0563c156c681a/hash/coleco_homebrew.xml#L72